Wikipedia/Code/TalkPageViewController.swift

```diff
@@ -339,6 +365,56 @@ extension TalkPageViewController: TalkPageReplyComposeDelegate {
     }
 }
 
+extension TalkPageViewController: TalkPageTopicComposeViewControllerDelegate {
+    func tappedPublish(topicTitle: String, topicBody: String, composeViewController: TalkPageTopicComposeViewController) {
+
+        viewModel.postTopic(topicTitle: topicTitle, topicBody: topicBody) { [weak self] result in
+
+            switch result {
+            case .success:
+                
+                composeViewController.dismiss(animated: true) {
+                    // TODO: Display success banner
+                }
+                
+                // Try to refresh page
+                self?.viewModel.fetchTalkPage { [weak self] result in
+                    switch result {
+                    case .success:
+                        self?.talkPageView.collectionView.reloadData()
+                        self?.scrollToLastTopic()
+                        self?.handleNewTopicAlert()
```

Comment:
For some reason, this banner does not show consistently when adding topics. I didn't notice any constraint errors on the console; it's not clear why it's behaving like this. I don't think it's a deal breaker for user testing, but it's worth looking at next.

Reply:
Thanks! I was seeing that too, I'll work on it in a follow up PR.